examples/blog/fakerest-init.js

```diff
@@ -0,0 +1,31 @@
+(function () {
+    'use strict';
+
+    // setup fake server
+    var restServer = new FakeRest.Server('http://localhost:3000');
+    var testEnv = window.location.pathname.indexOf('test.html') !== -1;
+    restServer.init(apiData);
```

Comment:
Nevermind, I saw what you loaded the data in a `<script>` tag before fakerest in index.html.